Automattic/jetpack
# Social: Update Nextdoor icon styling so it's round.

This is a quick fix to update the styling of the Nextdoor connection icon, so that it's displayed as round, rather than on a white square.





### Other information:


## Does this pull request change what data or activity we track or use?
No

## Testing instructions:






* Using D125910-code, create a Nextdoor connection
* In the editor sidebar notice how the icon is rendered

#### Before
<img width="201" alt="image" src="https://github.com/Automattic/jetpack/assets/96462/1572145c-212f-44ba-82d1-b9c6027b7455">

#### After
<img width="201" alt="image" src="https://github.com/Automattic/jetpack/assets/96462/3574867b-1340-4902-bdcf-88ff418f63d0">
